oat-sa/extension-tao-itemqti
# [Fix] MS-756 - remove fixed height for inline interaction in review mode

# Pull request summary

Fix UI bug for inline interaction in review mode

# How to test

1.   Create delivery with text inline interaction (item example attached)
[i_09_atviras_1603180625.zip](https://github.com/oat-sa/extension-tao-itemqti/files/5703938/i_09_atviras_1603180625.zip)
2.   Pass the test as TT
3.   As a scorer move to Task screen
4.   Check that text in inline interaction is vertically aligned
5.   Highlight some text, check that it stays within the border of the cell

# Checklist

- [ ] Pull request's target is not master
